lib/full-site-editing/block-templates.php

```diff
@@ -239,6 +312,15 @@ function gutenberg_get_block_templates( $query = array(), $template_type = 'wp_t
 		),
 	);
 
+	if ( 'wp_template_part' === $template_type && isset( $query['section'] ) ) {
+		$wp_query_args['tax_query'][]           = array(
```

Comment:
Aligning `=` looks off here. :)

Reply:
Composer formatted that way to line up with the `relation` defined just below.

Reply:
I figured, some of its auto-formatting is just weird. :D